Main/src/Coffee/Action.java

```diff
@@ -0,0 +1,165 @@
+package Coffee;
+
+import java.util.Scanner;
+
+public class Action {
+    public static int water=400;
+    public static int milk=540;
+    public static int coffeeBeans=120;
+    public static int dcups=9;
+    public static int money=550;
+
+    public static void getAction() {
+        System.out.println("Step 1");
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please enter required action: 'buy', 'fill' or 'pull'");
+        String actionType = in.nextLine();
+        if (actionType.equals("fill")) {
+                Action.fillMachine();
+            } else {
+                if (actionType.equals("pull")) {
+                    Action.pull();
+                } else {
+                    Action.orderCoffee(in);
+                }
+            }
+        }
+
+    public static void orderCoffee(Scanner in){
+        System.out.println("Step Order");
+        System.out.println("Please input coffee type ");
+        System.out.println("1. Espresso");
+        System.out.println("2. Latte");
+        System.out.println("3. Cappuccino");
+        String coffeeType = in.nextLine();
+        System.out.println("Please input cups quantity ");
+        int cup = in.nextInt();
+        if (coffeeType.equals("Espresso"))
+        {
+            Action.espresso(cup);
+        } else {
+            if (coffeeType.equals("Latte"))
+            {
+                Action.latte(cup);
+            } else Action.cappuccino(cup);
+        }
+        in.close();
+    }
+
+        public static void fillMachine()
+    {  stepOne: ;
```

Comment:
@GayaneGulanyan why ?

Reply:
In process of break and labels using, just try, removed